### PR TITLE
Analytics page has a giant empty content

### LINKED ISSUE #317: Analytics page has a giant empty content

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

-   [x] Bug Fix

## Related issues

Resolve #317

### CODE REVIEW

**Review by tsumo — APPROVED** [review]

**Review by 9teen90nine — APPROVED** [review]

File: src/pages/analytics/hire-stream/[hireStream].tsx [hunk 1]
```diff
@@ -19,11 +16,5 @@ export const getServerSideProps = createGetServerSideProps({
 });
 
 export default ({ stringIds }: InferServerSideProps<typeof getServerSideProps>) => {
```
Comment (nyet-ty) [inline src/pages/analytics/hire-stream/[hireStream].tsx:18]: does `export default AnalyticsHireStreamPage`  not work?

**Review by nyet-ty — CHANGES_REQUESTED** [review]

File: src/components/analytics/HiringBySectionType.tsx [hunk 2]
```diff
@@ -23,9 +20,6 @@ export const HiringBySectionType = ({ hireStreamName }: HiringBySectionTypeProps
 
     return (
         <>
-            <Text size="xl" style={{ marginTop: 10, marginLeft: 40 }}>
-                {tr('Hiring by section type')}
```
Comment (tsumo) [inline src/components/analytics/HiringBySectionType.tsx:27]: Why delete the title? Multiple chart blocks could be placed on a single page?

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/components/analytics/HiringBySectionType.tsx (hunk 2 above)
Comment (Katrin-kudryash) [reply src/components/analytics/HiringBySectionType.tsx]: I fixed it

File: src/controllers/AnalyticsHireStreamPage.tsx [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
 import { Paths } from '../utils/paths';
+import { HiringBySectionType } from '../components/analytics/HiringBySectionType';
+import { GradesByInterviewer } from '../components/analytics/GradesByInterviewer';
+import { AnalyticsFilterContextProvider } from '../contexts/analytics-filter-context';
 
 import { tr } from './controllers.i18n';
 
 export type HireStreamPageProps = {
-    hireStreamName?: string;
+    stringIds: Record<'hireStream', string>;
```
[the remark this answers is not in the document]
Comment (Katrin-kudryash) [reply src/controllers/AnalyticsHireStreamPage.tsx:11]: because getServerSideProps passes stringIds

**Review by tsumo — APPROVED** [review]

**Review by nyet-ty — APPROVED** [review]

**Pull request merged**